.dockerignore

```diff
@@ -4,3 +4,4 @@
 build/bin
 statusd-data
 wnode-data
+**/*.test
```

Comment:
Do we have any files ending with `.test`? Or you mean `**/*_test.go`?

Reply:
those are compiled leftovers after `go test`